modules/integration/src/test/java/org/apache/fluo/integration/client/FluoAdminImplIT.java

```diff
@@ -184,4 +197,98 @@ public void testInitializeLongChroot() throws Exception {
     }
   }
 
+
+  @Test
+  public void testRemove() throws Exception {
+
+    // the oracle server is started before every test so it is running
+    // double check by making sure its leader
+
+    try (FluoAdmin admin = new FluoAdminImpl(config)) {
+      admin.remove();
+      fail("expected remove() to fail with oracle server running");
+    } catch (FluoException e) {
+    }
+
+    oserver.stop();
+
+    try (CuratorFramework curator = CuratorUtil.newAppCurator(config)) {
+      curator.start();
+      Assert.assertFalse(OracleServerUtils.oracleExists(curator));
+    }
+    // this should succeed now with the oracle server stopped
+    try (FluoAdmin admin = new FluoAdminImpl(config)) {
+      admin.remove();
+    }
+
+    // verify path is not in zookeeper after remove
+    try (CuratorFramework curator = CuratorUtil.newRootFluoCurator(config)) {
+      curator.start();
+      String appRootDir = ZookeeperUtil.parseRoot(config.getAppZookeepers());
+      assertFalse(CuratorUtil.pathExist(curator, appRootDir));
+    }
+
+    // should succeed without clearing anything
+    try (FluoAdmin admin = new FluoAdminImpl(config)) {
+      InitializationOptions opts =
+          new InitializationOptions().setClearTable(false).setClearZookeeper(false);
+      admin.initialize(opts);
+    }
+
+    oserver.start();
+
+    // write some data into the table and test remove again
```

Comment:
It would be good to use the Fluo API to read and write data because if for some reason Fluo API calls fail after remove, we want to know.  Later in the test, I think it would still be good to use an Accumulo scanner to ensure the table is empty though.

Reply:
This test is throwing IllegalArgumentException after I try to do this using the fluo api.

> Received timestamp request with a Fluo application ID [a600fa56-cede-48f0-a06a-5cf41486c119] that does not match the application ID [838c2e17-b9c8-4a06-a8e9-9fcd79e0f9d5] of the Oracle